modules/gdscript/gdscript_parser.cpp

```diff
@@ -137,6 +137,68 @@ bool GDScriptParser::_enter_indent_block(BlockNode *p_block) {
 	}
 }
 
+bool GDScriptParser::_enter_inner_class_indent_block() {
+	if (tokenizer->get_token() != GDScriptTokenizer::TK_COLON) {
+		// report location at the previous token (on the previous line)
```

Comment:
Comments should start with a capital letter and end with a period (`.`).

```suggestion
		// Report location at the previous token (on the previous line).
```